docs/npm_nodejs.md

```diff
@@ -0,0 +1,59 @@
+# Using npm/Node.js code
+
+While Deno is pretty powerful itself, there is a large eco-system of code in the
+[npm](https://npmjs.com/) registry, and many people will want to re-leverage
+tools, code and libraries that are built for [Node.js](https://nodejs.org/). In
+this chapter we will explore how to use it.
+
+The good news, is that in many cases, it _just works_.
+
+There are some foundational things to understand about differences between
+Node.js and Deno that can help in understanding that challenges might be faced:
+
+- Current Node.js supports both CommonJS and ES Modules, while Deno only
+  supports ES Modules. The addition of stabilized ES Modules in Node.js is
+  relatively recent and most code written for Node.js is in the CommonJS format.
+- Node.js has quite a few built-in modules that can be imported and they are a
+  fairly expansive set of APIs. Deno has no built-in modules and only provides a
+  single global namespace/variable named `Deno` and it is part of our goals to
+  try to keep the APIs there to a minimum. Lots of code written for Node.js
```

Comment:
> Deno has no built-in modules and only provides a  single global namespace/variable named `Deno` and it is part of our goals to  try to keep the APIs there to a minimum.

Maybe something like "Deno instead provides browser compatible APIs where possible. Functionality that is needed beyond browser APIs is exposed in the `Deno` global variable."